src/runtime/web/index.js

```diff
@@ -1,6 +1,35 @@
-import { createSimpleContext } from './api';
-import { View } from '../../bindings/html/view';
+import { prioritizeLocales } from '../../intl/index';
+import { HTMLLocalization } from '../../bindings/html';
+import { documentReady, getResourceLinks, getMeta } from './util';
```

Comment:
so, `getResourceLinks` and `getMeta` will be different for HTML and XUL. Before your patch we have `bindings/html/head` and we can add `bindings/xul/head`. Not sure how you want to handle that here.